core/templates/pages/exploration-player-page/exploration-player-page.component.html

```diff
@@ -1,5 +1,8 @@
 <div>
-  <background-banner></background-banner>
+  <div *ngIf="isLoadingExploration" class="loader-container">
+    <oppia-loading-message [message]="'Loagding...'"></oppia-loading-message>
```

Comment:
`Loading` is spelled incorrectly but it is still spelled correctly in the video. Might wanna check this

Reply:
You can also check how other components in the repo are using this component. Can you please add a proof in different languages as well? For e.g. arabic?

Reply:
Hey @kapilag96, sorry, I pushed it by mistake...
I was checking if wrong spelled `loading` message is also showing or not.
(To see if the component is rendering correctly.)

Here is the proof of wrong spelled `loading` message video.

https://github.com/oppia/oppia/assets/108923011/53444eff-914b-4cf2-9a4e-40d1599ab6f5

I'll just push the correct code... : )

Reply:
Can you please add a video in a arabic and french? I see other uses of this component within the repo that rely on a service to get the loading message. Can you please check why?